test/e2e/load.go

```diff
@@ -36,14 +38,16 @@ const (
 	smallRCSize       = 5
 	mediumRCSize      = 30
 	bigRCSize         = 250
-	smallRCGroupName  = "load-test-small-rc"
-	mediumRCGroupName = "load-test-medium-rc"
-	bigRCGroupName    = "load-test-big-rc"
+	smallRCGroupName  = "load-small-rc"
+	mediumRCGroupName = "load-medium-rc"
+	bigRCGroupName    = "load-big-rc"
 	smallRCBatchSize  = 30
 	mediumRCBatchSize = 5
 	bigRCBatchSize    = 1
 )
 
+var createServices = os.Getenv("CREATE_SERVICES")
```

Comment:
I would prefer avoid having it as a global variable. Can you please move it to the test?